openwisp_monitoring/device/templates/admin/config/device/change_form.html

```diff
@@ -333,8 +333,10 @@ <h2>{% trans 'Interface status' %}: {{ interface.name }}</h2>
                   {% if interface.bridge_members %}
                   <div class="form-row">
                       <label>{% trans 'Bridge Members' %}:</label>
-                      <div class="readonly">
-                          {{ interface.bridge_members|join:", " }}
+                      <div class="readonly" >
+                          {% for element in interface.bridge_members %}
+                            <span class="bridgeMember">{{element}}</span>, 
```

Comment:
Please make this clickable by using an <a> element, use bridge-member as class name for consistency and also the variable syntax need to be consistent with the rest of the document, so `{{ element }}` instead of `{{element}`, please put effort in maintaining consistency, look around the code you write to make sure your code stays consistent.

Reply:
Yeah! I understand this, I will make sure to take care of this for sure and follow to make code consistent, will improve this.